### PR TITLE
Feature: Updated the Status Bar design

### PR DESCRIPTION
**Pull request by yaira2**
**Resolved / Related Issues**

To prevent extra work, all changes to the Files codebase must link to an approved issue marked as `Ready to build`. Please insert the issue number following the hashtag with the issue number that this Pull Request resolves.
- Updated the Status Bar design 
- This is prep work for completely floating panes

**Steps used to test these changes**

Stability is a top priority for Files and all changes are required to go through testing before being merged into the repo. Please include a list of steps that you used to test this PR.

1. Confirmed the corner radius is set to `0` when the sidebar is collapsed
2. Confirmed the sidebar resizer height matches the border height
3. Confirmed the border thickness is correct when Dual Pane mode is enabled
4. Confirmed the border thickness is correct when the Info Pane is opened

**Screenshots**
image

### CODE REVIEW

File: src/Files.App/Views/PaneHolderPage.xaml [hunk 1]
```diff
@@ -126,7 +126,7 @@
 				<icore:ChangePropertyAction
 					PropertyName="CurrentInstanceBorderThickness"
 					TargetObject="{Binding ElementName=PaneLeft}"
-					Value="0,0,0,1" />
+					Value="0,0,0,0" />
```
Comment (marcelwgn) [inline src/Files.App/Views/PaneHolderPage.xaml:129]: Is this related and if so, how does this affect the Statusbar?

File: src/Files.App/Views/MainPage.xaml [hunk 2]
```diff
@@ -215,78 +215,81 @@
...
+				</Grid>
+			</sidebar:SidebarView.InnerContent>
 
+			<!--  Footer Content  -->
+			<sidebar:SidebarView.FooterContent>
+				<Border>
+					<!--  Status Bar  -->
+					<uc:StatusBar
+						x:Name="StatusBar"
+						Height="32"
```
Comment (marcelwgn) [inline src/Files.App/Views/MainPage.xaml:288]: Could this be a problem with larger text sizes and text getting clipped? (Not like this wouldn't have been an issue without this PR)

**Review by marcelwgn — APPROVED** [review]
LGTM. I'm a bit concerned that if all items are expanded, there is no visual separator between the footer content and the sidebar items, but I think that is quite hard to solve. Overall, I think it looks quite well though.

Besides that, I think its good. I'm not too happy with the name FooterContent (since Footer is also something one could expect in the left pane) but I also can't think of anything more fitting.

**Comment by 0x5bfa** [comment]
Yair is going to add Settings button as a pane footer.

image

As I agree with @marcelwgn on that, it should be renamed to 'ContentFooter' and pane one to be 'PaneFooter'.

File: src/Files.App/Views/PaneHolderPage.xaml (hunk 1 above)
Comment (yaira2) [reply src/Files.App/Views/PaneHolderPage.xaml:129]: It’s an unrelated fix, I can try to send the before and after…

File: src/Files.App/Views/MainPage.xaml (hunk 2 above)
Comment (yaira2) [reply src/Files.App/Views/MainPage.xaml:288]: I haven’t seen any reports of this nature, but like you said, it wouldn’t be a new issue if that’s the case…

File: src/Files.App/Views/PaneHolderPage.xaml (hunk 1 above)
Comment (marcelwgn) [reply src/Files.App/Views/PaneHolderPage.xaml:129]: Just wanted to make sure, I understand this change correctly and that I didnt miss anything.

Comment (yaira2) [reply src/Files.App/Views/PaneHolderPage.xaml:129]: Actually, this change is related.

The border is handled on a different control now so we no longer apply it here.

**Pull request merged**